projects/go-style-guide/src/app/features/ui-kit/components/form-docs/components/select-docs/select-docs.component.html

```diff
@@ -487,7 +487,8 @@ <h4 class="go-heading-4 go-heading--underlined">View</h4>
               [items]="options$ | async"
               [multiple]="true"
               label="Your Input"
-              [typeahead]="itemInput">
+              [typeahead]="itemInput"
+              [typeToSearchText]="'Search The Thing'">
```

Comment:
Since this is just a string can we change this to `typeToSearchText="Search The Thing"`?

Reply:
Fair enough, I switched it.